sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -585,6 +589,12 @@ public DigitalTwinsServiceVersion getServiceVersion() {
             nextLink -> withContext(context -> listRelationshipsNextPage(nextLink, context)));
     }
 
+    PagedFlux<String> listRelationships(String digitalTwinId, String relationshipName, Context context) {
```

Comment:
This function was sitting in the models API section, so I moved it back to the correct section